tel2puml/__main__.py

```diff
@@ -252,17 +284,77 @@ def generate_component_options(
     return otel_pv_options, pv_puml_options
 
 
+def handle_exception(
+    e: Exception,
+    debug: bool,
+    user_error: bool = False,
+    custom_message: str = "",
+) -> None:
+    """Handle exceptions with custom messaging.
+
+    :param e: The exception instance to handle.
+    :type e: :class:`Exception`
+    :param debug: Flag to indicate if debug information should be printed.
+    :type debug: `bool`
+    :param user_error: Flag to indicate if the error is a user error,
+    defaults to False.
+    :type user_error: `bool`, optional
+    :param custom_message: Custom error message for user, defaults to "".
+    :type custom_message: `str`, optional
+    """
+    if debug:
+        print(f"DEBUG: {traceback.format_exc()}")
+    else:
+        print("\nERROR: Use the -d flag for more detailed information.")
+        if user_error:
+            print(f"\nUser error: {custom_message}")
+            print(f"\n{e}")
+        else:
+            print("\nAn unexpected error occurred")
+            print(f"\n{e}")
+            print("\nPlease contact smartDCSIT support for assistance.")
```

Comment:
Can we change this to "Please raise an issue at https://github.com/xtuml/otel2puml"